crates/hir/src/code_model.rs

```diff
@@ -873,6 +873,14 @@ impl SelfParam {
             })
             .unwrap_or(Access::Owned)
     }
+
+    pub fn display(self, db: &dyn HirDatabase) -> &'static str {
```

Comment:
I'm not sure if it is a good idea to put this here, or if I should just inline it (or put it somewhere else)?